recipes/libsimpleitk/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# When building 32-bits on 64-bit system this flags is not automatically set by conda-build
+if [ $ARCH == 32 -a "${OSX_ARCH:-notosx}" == "notosx" ]; then
+    export CFLAGS="${CFLAGS} -m32"
+    export CXXFLAGS="${CXXFLAGS} -m32"
+fi
+
+BUILD_DIR=${SRC_DIR}/build
+mkdir ${BUILD_DIR}
+cd ${BUILD_DIR}
+
+cmake \
+    -G Ninja \
+    -D "CMAKE_CXX_FLAGS:STRING=${CXXFLAGS}" \
+    -D "CMAKE_C_FLAGS:STRING=${CFLAGS}" \
+    ${CMAKE_ARGS} \
+    -D CMAKE_EXE_LINKER_FLAGS:STRING=-fuse-ld=gold \
+    -D CMAKE_MODULE_LINKER_FLAGS:STRING=-fuse-ld=gold \
+    -D CMAKE_SHARED_LINKER_FLAGS:STRING=-fuse-ld=gold \
```

Comment:
From what I have seen this might be a bug with `clang`. Though the result is the same.